app/controllers/katello/api/v2/content_exports_controller.rb

```diff
@@ -0,0 +1,58 @@
+module Katello
+  class Api::V2::ContentExportsController < Api::V2::ApiController
+    before_action :find_exportable_content_view_version, :only => [:version]
+
+    api :GET, "/content_exports/index", N_("Show the export history for a content view version")
```

Comment:
This route should be just `/content_exports` (see other `index` methods)

Reply:
Fixed